clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/ResizeAsgAtomicOperationUnitSpec.groovy

```diff
@@ -110,8 +111,65 @@ class ResizeAsgAtomicOperationUnitSpec extends Specification {
     capacity(0, 2, 3)    | 1          | 2          | 3              || true
   }
 
+  @Unroll
+  void "should support resize requests with only some fields to update (min: #minSize, max: #maxSize, desired: #desiredCapacity)"() {
+    setup:
+    def description = new ResizeAsgDescription(
+      asgs: [[
+               serverGroupName: "myasg-stack-v000",
+               region         : "us-west-1",
+               capacity       : new ServerGroup.Capacity(min: minSize, max: maxSize, desired: desiredCapacity)
```

Comment:
IMO I'd probably be consistent on the naming ... minSize vs maxSize vs desiredCapacity.

Reply:
Ha, I know. I am sticking to the AWS terminology though

Reply:
Reasonable argument I suppose

Reply:
Reasonable argument I suppose